library/X509/Job.php

```diff
@@ -203,6 +206,16 @@ function (ConnectionInterface $conn) use ($target, $streamCapture) {
                 // Close connection in order to capture stream context options
                 $conn->close();
 
+                $this->db->update(
+                    'x509_target',
+                    ['last_seen' => $this->currentDateTime],
```

Comment:
If you use something like this and you wouldn’t need `$this->currentDateTime` at all.
```php
['last_seen' = new Expression('UNIX_TIMESTAMP()')]
```